src/tokens.ts

```diff
@@ -25,7 +24,7 @@ export class TokenComputer {
     constructor() {}
 
     isFungible(token: Token): boolean {
-        return token.nonce === 0;
+        return token.nonce == 0n;
```

Comment:
Is there any reason to not use `===`?

Reply:
Helps a bit if someone uses sdk-core from pure JavaScript, and aren't constrained (by TS) to pass a `bigint` instead of a `number`. E.g.

```
> BigInt(0) == 0
true
> BigInt(0) === 0
false
```

But fixed, using `===` now.